xun/tests/helpers.py

```diff
@@ -121,6 +121,13 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         return exc_type is None
 
 
+def run_in_process(blueprint):
+    return blueprint.run(
+        driver=xun.functions.driver.Sequential(),
+        store=xun.functions.store.Memory()
+    )
```

Comment:
I don't quite understand the function name

Reply:
It's a sentance, you have an action (run_in_process), that you apply to "blueprint"

Reply:
Okey, fair enough